IOPool/Input/scripts/makeFileContainingStreamerInfos.C

```diff
@@ -0,0 +1,90 @@
+// Build a working release including the desired class versions in the
+// class definitions and classes_def.xml files. You might need to add or
+// remove lines below to get the StreamerInfo objects for the desired classes.
+// Then run the following to execute this script:
+//
+//   root -l -b -q makeFileContainingStreamerInfos.C
+//
+// Then rename the output file as appropriate. If it is of general use
+// you might want to reposit the file in the IOPool/Input data repository.
+// This output file can be used by the service named FixMissingStreamerInfos.
+
+#include <iostream>
+
+void makeFileContainingStreamerInfos() {
+  std::cout << "Executing makeFixitFile()" << std::endl;
```

Comment:
The name in the printout is different from the function and file name.

Reply:
I fixed that so the printout and names match. Thanks.